example/src/lint.ts

```diff
@@ -0,0 +1,3 @@
+// this is a linting violation
+const lint: string = "warning";
```

Comment:
[ESLint] Type string trivially inferred from a string literal, remove type annotation  [@typescript-eslint/no-inferrable-types]

```suggestion
const lint = "warning";
```